autozone-poc/src/components/Signup/Signup.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import $ from 'jquery';
+import axios from "axios";
+import InputWithLabel from "../../utlis/formComponents/InputWithLabel";
+import "./Signup.css";
+import Button from "../../utlis/button/Button";
+
+//export default class Signin extends Component{
+
+  export default function Signup (props) {
+  // const {register, handleSubmit, errors, watch}=useForm();
+  const [firstname, setFirstName] = useState("");
+  const [lastname, setLastName] = useState("");
+  const [emailid, setEmailid] = useState("");
+  const [password, setPasswords] = useState("");
+  const [confirmpassword, setConfirmPassword] = useState("");
+
+  $(document).ready(function(){
```

Comment:
remove jquery validation and implement state-based validation in react

Reply:
Validation already written in j-query for reusable component, that why i written in j-query to achieve scenario